guillotina/fields/annotation.py

```diff
@@ -309,6 +312,13 @@ def get_annotation_name(self, bucket_id: str) -> str:
         bucket["len"] = len(annotation["keys"])
         annotation.register()
 
+    async def clear(self, context):
```

Comment:
This will return all the annotations on the object.

What you want to do is something like this:

```
for bk in self.buckets:
  annotation_name = self.get_annotation_name(bk["id"])
  await annotations_container.async_del(annotation_name)
```